tkupchyn/homework_03/the_most_wanted_letter.py

```diff
@@ -22,3 +22,8 @@ def find_most_frequent_letter(phrase: str) -> str:
             letter_frequency[symbol] = 1
 
     return min([elem for elem in letter_frequency if letter_frequency[elem] == max(letter_frequency.values())])
+
+
+def find_most_frequent_letter_alternative(phrase: str) -> str:
+    phrase = [symbol.lower() for symbol in phrase if symbol.isalpha()]
+    return max([elem for elem in reversed(phrase)], key=phrase.count)
```

Comment:
```
from string import asci_lower

def find_most_frequent_letter_alternative(phrase: str) -> str:
      return max(asci_lower, key=phrase.lower().count)

```

Reply:
you need to find lower letter, priority alphabet, so you just search max letter among alphabet letters based on their count in phrase.lower()